Docs/source/acknowledge.rst

```diff
@@ -0,0 +1,24 @@
+Acknowledge WarpX
+=================
+
+Please acknowledge the role that WarpX played in your research by adding the following sentence to your publications, it helps contributors keep track of users and promote the project:
+
+**Plain text:**
+
+  This research used the open-source particle-in-cell code WarpX https://github.com/ECP-WarpX/WarpX, primarily funded by the US DOE Exascale Computing Project. We acknowledge all WarpX contributors.
+
+**Latex:**
+
+.. code-block:: latex
+
+  \usepackage{hyperref}
+  This research used the open-source particle-in-cell code WarpX \url{https://github.com/ECP-WarpX/WarpX}, primarily funded by the US DOE Exascale Computing Project. We acknowledge all WarpX contributors.
+
+**Main WarpX reference:**
+
+.. raw:: html
+
+    <div id="ref-VayAAC2010">
+
+Vay, J -. L, C G R Geddes, C Benedetti, D L Bruhwiler, E Cormier-Michel, B M Cowan, J R Cary, and D P Grote. 2010. “Modeling Laser Wakefield Accelerators in A Lorentz Boosted Frame.” *Aip Conference Proceedings* 1299: 244–49. https://doi.org/10.1063/1.3520322.
```

Comment:
Isn't the last EAAC paper (see readme) our current go-to reference? :)

Reply:
Oops, I copy-pasted the reference just for formatting, and forgot to replace it with the actual paper..